terraform/service/oqtopus-dev/main.tf

```diff
@@ -55,6 +57,13 @@ module "user_api" {
   allow_methods                 = "GET,POST,PUT,PATCH,DELETE"
   allow_headers                 = "Content-type,Accept,Authorization,Q-API-Token"
   log_level                     = "INFO"
+
```

Comment:
These values are retrieved  by actual Lambda configuration.
These magic numbers should be replaced by appropriate references to other resources' outputs.